R/axis_geo.R

```diff
@@ -49,33 +47,39 @@
+#'
 #' \code{axis_geo_phylo(...)} is shorthand for
 #' \code{axis_geo(..., phylo = TRUE)}.
 #'
 #' @param side \code{integer}. Which side to add the axis to (\code{1}: bottom,
 #'   the default; \code{2}: left; \code{3}: top; \code{4}: right).
 #' @param intervals The interval information to use to plot the axis: either A)
-#'   a \code{character} string indicating a built-in or remotely hosted
-#'   \code{data.frame} (see \code{\link[deeptime]{get_scale_data}}), or B) a
+#'   a \code{character} string indicating a rank of intervals from the built-in
+#'   \code{\link{GTS2020}}, B) a \code{character} string indicating a remotely
+#'   hosted \code{data.frame} (see \code{\link{get_timescale_data}}), or C) a
 #'   custom \code{data.frame} of time interval boundaries (see Details).
 #' @param height \code{numeric}. The relative height (or width if \code{side} is
 #'   \code{2} or \code{4}) of the scale. This is relative to the height (if
 #'   \code{side} is \code{1} or \code{3}) or width (if \code{side} is \code{2}
 #'   or \code{4}) of the plot.
 #' @param fill \code{character}. The fill color of the boxes. The default is to
-#'   use the \code{color} column included in \code{intervals}. If a custom
-#'   dataset is provided with \code{intervals} without a \code{color} column and
-#'   without specifying \code{fill}, a greyscale will be used. Custom fill
-#'   colors can be provided with this option (overriding the \code{color}
-#'   column) and will be recycled if/as necessary.
+#'   use the \code{colour} or \code{color} column included in \code{intervals}.
+#'   If a custom dataset is provided with \code{intervals} without a
+#'   \code{colour} or \code{color} column and without specifying \code{fill}, a
+#'   greyscale will be used. Custom fill colours can be provided with this
+#'   option (overriding the \code{colour} or \code{color} column) and will be
+#'   recycled if/as necessary.
 #' @param lab \code{logical}. Should interval labels be included?
-#' @param lab_col \code{character}. The color of the labels. The default is to
-#'   use the \code{lab_color} or \code{lab_colour} column included in
+#' @param lab_col \code{character}. The colour of the labels. The default is to
+#'   use the \code{font} or \code{lab_color} column included in
 #'   \code{intervals}. If a custom dataset is provided with \code{intervals}
-#'   without a \code{lab_color} or \code{lab_colour} column and without
-#'   specifying \code{lab_col}, all labels will be black. Custom label colors
-#'   can be provided with this option (overriding the \code{lab_color} or
-#'   \code{lab_colour} column) and will be recycled if/as necessary.
+#'   without a \code{font} or \code{lab_color} column and without
+#'   specifying \code{lab_col}, all labels will be black. Custom label colours
+#'   can be provided with this option (overriding the \code{font} or
+#'   \code{lab_color} column) and will be recycled if/as necessary.
```

Comment:
Does "color" need changing to "colour" here too?

Reply:
No, because that's referring to the deeptime standards

Reply:
"font" is the palaeoverse standard